src/operator/nn/mkldnn/mkldnn_act.cc

```diff
@@ -83,6 +84,8 @@ mkldnn::algorithm GetMKLDNNActAlgo(const ActivationParam& param) {
       return mkldnn::algorithm::eltwise_relu;
     case activation::kSigmoid:
       return mkldnn::algorithm::eltwise_logistic;
+    case activation::kLogSigmoid:
```

Comment:
This function is used with oneDNN subgraphs also, can you add tests in test_fc_subgraph.py and test_conv_subgraph.py

Reply:
Will do